qlib/workflow/recorder.py

```diff
@@ -227,18 +239,20 @@ def set_tags(self, **kwargs):
         else:
             mlflow.set_tags(dict(kwargs))
 
-    def delete_tag(self, key):
-        mlflow.delete_tag(key)
+    def delete_tags(self, *keys):
+        for count, key in enumerate(keys):

```

Comment:
All the objects and parameters are logged into a running Recorder. 
Some the recorder may not be a running recorder.